templates/.pre-commit-config.yaml.jinja

```diff
@@ -103,8 +103,7 @@ repos:
           - --max-warnings=0
         additional_dependencies:
           {% block eslint_dependencies %}
-          - eslint@8.15.0
-          - eslint-config-prettier@8.5.0
+          - eslint@8.28.0
```

Comment:
Why do we remove `eslint-config-prettier`?

Reply:
It's not removed, it was listed twice in the list with different versions so I deduped that. I have updated the template in 3857de0.